src/app/collections/Mod.tsx

```diff
@@ -123,16 +123,9 @@ export default function Mod({ item, defs, allowFilter, innerRef, onClick, childr
   }
 
   const modDef = defs.InventoryItem.get(item.hash);
-  if (!item) {
```

Comment:
is this not needed for exiting early?

Reply:
it already tests the same condition a couple lines up